test/testdata/simpletests/datavolume.json

```diff
@@ -0,0 +1,8 @@
+{
+  "TaskDefinition": "datavolume",
```

Comment:
I think what I don't see in the tasks is what the test is doing - take "datavolume2" for example - I like that it has "related", but I don't know what cases it intends to exercise. I think a readable description would be useful here - similar to a test_method_name or it description in rspec. Tags could be useful later (for filtering tests), but probably not required now.